src/chigame/api/tests/factories.py

```diff
@@ -1,5 +1,6 @@
 import random
 
+import factory
```

Comment:
Maybe for readibility sake, just have the imports into one line as in 
```
from factory import * 
```

Reply:
Will change now!

Reply:
in general it is a bad idea to import everything, since then it is much harder to tell what you are actually using from a given library.

It is better to do the work of writing out what you are doing (generally in alphabetical order) since then it is clear exactly what you are using from each file.